api/groups/v1_2/interface.go

```diff
@@ -0,0 +1,11 @@
+package v1_2
+
+import "github.com/ElrondNetwork/elrond-proxy-go/data"
+
+// AccountsFacadeHandlerV1_2 interface defines methods that can be used from facade context variable
+type AccountsFacadeHandlerV1_2 interface {
```

Comment:
Why do we need to have a single facade implementation that will satisfy all the interfaces?
The question arose from seeing the function `GetShardIDForAddressV1_2`.
We can have multiple facade implementations that will be consumed by various endpoint versions. Having all functions in a single monstrous facade implementation will impact struct size (which should be kept as small as possible)

Reply:
Do you mean that we should drop the `facade` entity that holds all the processors and calls the appropriate methods of them?